core/domain/feedback_services_test.py

```diff
@@ -81,34 +81,12 @@ def test_feedback_ids(self):
 
     def test_create_message_fails_if_invalid_thread_id(self):
```

Comment:
There should be more tests added to this file to reflect the code changes that were made in the corresponding code file.